params.ini

```diff
@@ -11,8 +11,11 @@ graphic_debugger_key=f10
 logg_lvl=debug
 randomize_runs=0
 difficulty=hell
-custom_discord_hook=
+custom_message_hook=
 discord_status_count=20
+message_headers=
+message_body_template={{"content": "{msg}"}}
+message_highlight=1
```

Comment:
Maybe these would be good at the [advanced_options] section. Most users already struggle creating a discord hook :D